Formula/argo.rb

```diff
@@ -0,0 +1,35 @@
+class Argo < Formula
+  desc "Get stuff done with container-native workflows for Kubernetes"
+  homepage "https://argoproj.io"
+  url "https://github.com/argoproj/argo.git",
+      :tag      => "v2.9.3",
+      :revision => "9407e19b3a1c61ad4043e382484fd0b6b15574f2"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+  depends_on "yarn" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
Does this not work with go modules?

Reply:
Yes, after removing `GOPATH` etc. the same problem remains.